addons/godot-firebase/firebase/firebase.gd

```diff
@@ -1,8 +1,8 @@
 # ---------------------------------------------------- #
-#                 SCRIPT VERSION = 2.3                 #
+#                 SCRIPT VERSION = 2.4                 #
 #                 ====================                 #
 # please, remember to increment the version to +0.1    #
-# if you are going to make changes that will commited  #
+# if you are going to make changes that will committed #
```

Comment:
Thanks. :D

Reply:
I do it all the time. I had to install a spell check in VSCode